ublox/Cargo.toml

```
@@ -4,7 +4,7 @@ description = "A crate to communicate with u-blox GPS devices using the UBX prot
 name = "ublox"
 homepage = "https://github.com/ublox-rs"
 repository = "https://github.com/ublox-rs/ublox"
-version = "0.8.0"
+version = "0.9.0"
```

We will bump the version in another PR when we cut a new release

```suggestion
version = "0.8.0"
```

---

This was to address the semver CI jobs that were failing because adding new variants to the public PacketRef enum is considered a breaking change.

I bumped it to 0.9.0 just to get that CI check to pass. Is there a way to suppress or ignore the SemVer check for this PR, or should I leave the version at 0.8.0 and let that specific check fail?